gc.rb

```diff
@@ -130,33 +130,59 @@ def self.count
-  #          :minor_gc_count=>0,
-  #          :major_gc_count=>0,
-  #          :remembered_wb_unprotected_objects=>0,
-  #          :remembered_wb_unprotected_objects_limit=>0,
-  #          :old_objects=>0,
-  #          :old_objects_limit=>0,
-  #          :oldmalloc_increase_bytes=>2389760,
-  #          :oldmalloc_increase_bytes_limit=>16777216
-  #      }
+  #  [count]
+  #    The total number of garbage collections run since process start,
+  #    including minor and major garbage collections
+  #  [heap_allocated_pages]
+  #    The total number of `:heap_eden_pages` + `:heap_tomb_pages`
+  #  [heap_sorted_length]
+  #    The number of pages that can fit into memory between the first and last
+  #    page
+  #  [heap_allocatable_pages]
+  #    The total number of pages the process could allocate without additional GC
+  #  [heap_available_slots]
+  #    The total number of slots in all `:heap_allocated_pages`
+  #  [heap_live_slots]
+  #    The total number of slots which contain live objects
+  #  [heap_free_slots]
+  #    The total number of slots which do not contain live objects
+  #  [heap_final_slots]
+  #    The total number of slots with finalizers
+  #  [heap_marked_slots]
+  #    The total number of old objects and write barrier unprotected objects
+  #  [heap_eden_pages]
+  #    The total number of pages which contain at least one live slot
+  #  [heap_tomb_pages]
+  #    The total number of pages which do not contain any live slots
+  #  [total_allocated_pages]
+  #    The cumulative number of pages allocated since process start
+  #  [total_freed_pages]
+  #    The cumulative number of pages freed since process start
+  #  [total_allocated_objects]
+  #    The cumulative number of objects allocated since process start
+  #  [total_freed_objects]
+  #    The cumulative number of objects freed since process start
+  #  [malloc_increase_bytes]
+  #    Amount of memory allocated on heap for live, young objects
+  #  [malloc_increase_bytes_limit]
+  #    When `:malloc_increase_bytes` crosses this limit, minor GC is triggered
+  #  [minor_gc_count]
+  #    The total number of minor garbage collections run since process start
+  #  [major_gc_count]
+  #    The total number of major garbage collections run since process start
+  #  [remembered_wb_unprotected_objects]
+  #    The total number of objects without write barriers
+  #  [remembered_wb_unprotected_objects_limit]
+  #    When `:remembered_wb_unprotected_objects` crosses this limit,
+  #    major GC is triggered
+  #  [old_objects]
+  #    Number of live, old objects which have survived at least 3 garbage collections
+  #  [old_objects_limit]
+  #    When `:old_objects` crosses this limit, major GC is triggered
+  #  [oldmalloc_increase_bytes]
+  #    Amount of memory allocated on heap for live, old objects
+  #  [oldmalloc_increase_bytes_limit]
+  #    When `:old_malloc_increase_bytes` crosses this limit, major GC is triggered
```

Comment:
This is similar to the `malloc_increase_bytes` above, except this value triggers a major GC instead of any GC.